Lib/test/test_funcattrs.py

```diff
@@ -453,6 +453,29 @@ class BuiltinFunctionPropertiesTest(unittest.TestCase):
     # XXX Not sure where this should really go since I can't find a
     # test module specifically for builtin_function_or_method.
 
+    def test_builtin__module__(self):
+        import decimal
+        import math
+
+        # builtin function:
+        self.assertEqual(len.__module__, 'builtins')
+        self.assertEqual(math.sin.__module__, 'math')
+
+        # instance method:
+        self.assertRaises(AttributeError, getattr, int.to_bytes, '__module__')
+        self.assertRaises(AttributeError, getattr, decimal.Decimal.exp, '__module__')
+        self.assertEqual(int.to_bytes.__objclass__.__module__, 'builtins')
+        self.assertEqual(decimal.Decimal.exp.__objclass__.__module__, 'decimal')
+
+        # builtin classmethod:
+        self.assertEqual(int.from_bytes.__module__, None)
+        self.assertEqual(int.from_bytes.__self__.__module__, 'builtins')
```

Comment:
Note that for classmethods the ``__module__`` attribute is also empty.  We could extend proposed solution to cover this case as well.  But this looks as a less severe issue for me: the module value is available in this case via the ``__self__`` attribute.